internal/pkg/service/buffer/store/export.go

```diff
@@ -53,18 +53,22 @@ func (s *Store) UpdateExport(ctx context.Context, export model.Export) (err erro
 	_, span := s.tracer.Start(ctx, "keboola.go.buffer.configstore.UpdateExport")
 	defer telemetry.EndSpan(span, &err)
 
-	err = s.DeleteExport(ctx, export.ExportKey)
-	if err != nil {
-		return err
-	}
-	err = s.CreateExport(ctx, export)
-	if err != nil {
-		return err
-	}
+	_, err = op.MergeToTxn(
+		s.updateExportBaseOp(ctx, export.ExportBase),
+		s.updateMappingOp(ctx, export.Mapping),
+	).Do(ctx, s.client)
```

Comment:
Aha, tady je. 😄 Tak super. 👍

Reply:
Jo, to se upravilo až v pozdějším commitu. Sorry za zmatek 🙂